components/Location/Location.js

```diff
@@ -132,10 +132,12 @@ const getDailySchedules = (schedules) => {
     .forEach(schedule => {
       const day = schedule.weekday.toLowerCase()
       const daySchedule = daySchedules[day]
```

Comment:
could wrap these variable definitions in the `if` to save a little work in creating them if they aren't used.